doc/rpm2archive.8

```diff
@@ -0,0 +1,30 @@
+.\" rpm2cpio - Converts Red Hat Package (RPM) to tar archive
+.TH RPM2ARCHIVE 8 "27 January 2020" "Red Hat, Inc."
+.SH NAME
+rpm2archive \- Create tar archive from RPM Package Manager (RPM) package.
+.SH SYNOPSIS
+\fBrpm2archive\fP [filename] 
+.SH DESCRIPTION
+\fBrpm2archive\fP converts the .rpm files specified as arguments to gz
+compressed tar files with suffix ".tgz".
+
+If '-' is given as argument, an rpm stream is read from standard in and
+written to standard out.
+
+In opposite to \fBrpm2cpio\fP \fBrpm2archive\fP also works with RPM packages
+containing files greater than 4GB which are not supported by cpio. Unless
+\fBrpm2cpio\fP \fBrpm2archive\fP needs a working rpm installation which limits
+its usefulness for some disaster recovery scenarios.
+
+.br
+.I "\fBrpm2archive glint-1.0-1.i386.rpm ; tar -xvz glint-1.0-1.i386.rpm.tgz\fP"
+.br
+.I "\fBcat glint-1.0-1.i386.rpm | rpm2cpio - | tar -tvz\fP"
```

Comment:
Shouldn't this be `rpm2archive` instead?

Reply:
Yes, yes, it should.